Specifications/QIR/Quantum-Runtime.md

```diff
@@ -14,9 +14,9 @@ We define the following functions for managing qubits:
 | Function                            | Signature       | Description |
 |-------------------------------------|-----------------|-------------|
 | __quantum__rt__qubit_allocate       | `%Qubit*()`     | Allocates a single qubit. |
-| __quantum__rt__qubit_allocate_array | `%Array*(i64)`  | Allocates an array of qubits. |
+| __quantum__rt__qubit_allocate_array | `%Array*(i64)`  | Creates an array of the given size and populates it with newly allocated qubits. |
 | __quantum__rt__qubit_release        | `void(%Qubit*)` | Release a single qubit. |
-| __quantum__rt__qubit_release_array  | `void(%Array*)` | Release an array of qubits. |
+| __quantum__rt__qubit_release_array  | `void(%Array*)` | Release each qubit in the given array. The array itself is *not* unreferenced, and needs to be dereferenced separately for the memory to be freed. |
```

Comment:
Maybe rename to something like `__quantum__rt__qubit_release_all`?

Reply:
I reconsidered what that function should do and correspondingly keep the name.